src/test/scala/com/fulcrumgenomics/umi/CallDuplexConsensusReadsTest.scala

```diff
@@ -58,14 +59,24 @@ class CallDuplexConsensusReadsTest extends UnitSpec {
     checkClpAnnotations[CallDuplexConsensusReads]
   }
 
+  it should "run fail if AB-R1s are not on the same strand ads BA-R2s" in {
+    val builder = new SamRecordSetBuilder(readLength=10)
+    builder.addPair(name="ab1", start1=100, start2=100, attrs=Map(MI -> "1/A")).foreach { _.setReadString("AAAAAAAAAA") }
```

Comment:
It would be nice to change `start2` just so that it's a little more clear.  The test is valid, but having R1 and R2 starts be the same makes it harder to see that.